dentapp/src/components/IconButton.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { FaUserAlt } from "react-icons/fa";
+import { MdShoppingCart } from "react-icons/md";
+import Styled from "styled-components";
+import PropTypes from "prop-types";
+
+const darkgrey = "#4A4A4A";
+const cyan = " #00D9CC";
+
+const Icon = Styled.button`
+border-radius: ${(props) => (props.borderRadius ? true : false)};
+color: ${(props) => (props.color ? cyan : darkgrey)};
+font-size: ${(props) => (props.fontSize ? "16px" : "20px")}
+`;
+
+function IconButton() {
+  const { fontSize, borderRadius, color, onClick } = this.props;
+  return (
+    <Icon
+      onClick={onClick}
+      rounded={borderRadius}
+      color={color}
+      size={fontSize}
+      userIcon={<FaUserAlt />}
+      shoppingIcon={<MdShoppingCart />}
+    ></Icon>
+  );
+}
+
+export default IconButton;
+
+IconButton.propTypes = {
+  borderRadius: PropTypes.bool.isRequired,
```

Comment:
call this prop rounded instead of borderRadius

Reply:
And for what I understand is no required, so it will need a default value, use false

Reply:
User Icon finished

![image](https://user-images.githubusercontent.com/65319385/86312425-4d2fb380-bbe8-11ea-8c9f-e1dc4986f0e0.png)